src/golang/cmd/migrator/versions/000026_add_operator_artifact_node_views/up_sqlite.go

```diff
@@ -0,0 +1,141 @@
+package _000026_add_operator_artifact_node_views
+
+const upSqliteScript = `
```

Comment:
Views in SQLite are just stored SQL statements, so there isn't any performance benefit of having precomputed values ready to go. I feel like it may be more flexible to just define this as a regular query inside of a method in one of our repos instead of defining a database view for it. 

That way if we need to make changes to the query it can have no overhead as opposed to updating a view with another migration step.

Reply:
I see, would you suggest by pre-running these `CREATE VIEW` while executing the queries?

Reply:
In this case, do you see any draw-back if this view creation is running multiple times?

Reply:
I would remove the `CREATE VIEW` part completely and just run the query, because SQLite won't update the view when the underlying tables change: https://www.sqlite.org/lang_createview.html.

This is different from something like Postgres which has support for updateable views, but we don't support Postgres right now, so we should optimize for Postgres when we do add support for it: https://www.percona.com/blog/postgresql-updatable-views-performing-schema-updates-with-minimal-downtime/#:~:text=PostgreSQL%20has%20introduced%20automatically%20updatable,as%20on%20a%20regular%20table.

Reply:
I think the main thing is running `CREATE VIEW` doesn't add any value for us right now, and running the view creation multiple times is extra overhead for SQLite, as opposed to just running the query.

Reply:
Thanks! So should we just do `WITH (%s) AS operator_nodes` and put this in a helper?

Reply:
You can start from `WITH edge_ordered AS...` and keep everything else the same. I don't think you need to define `operator_nodes` since that is what you are returning

Reply:
Cool, this PR and #1165 should be ready!